RPIPs/RPIP-30.md

```diff
@@ -47,45 +47,77 @@ the protocol goal of attracting minipool creation (and rETH supplying).
 
 ## Specification
```

Comment:
https://github.com/Valdorff/RPIPs/blob/sandbox/RPIPs/RPIP-30.md#specification

Tried it out in a sandbox branch -- I think I like it better as is? Can't really get rid of the definitions fully, so it simplifies but also adds more stuff.

Reply:
I would move those definitions to the section where they are relevant, rather than having a separate definition section. I actually prefer the new version, so maybe it's just a personal thing?

Reply:
Eg:
`current_method_share` goes in transition section
Staked / Unstaked / Staking goes in withdrawal section
X=1 goes in transition section

Starting to think I may have misunderstood Staked/ Unstaked / Staking as not actually connecting to other sections, and being just general definitions? Not sure.

Reply:
My issue is that while `current_method_share` has a clear home in the transition section, `proposed_method_share` is relevant to both "proposed reward calculation" and its attendant transition section. That type of thing leads me to prefer the separate definition section